modules/core/src/v2/internal/tssUtils.ts

```diff
@@ -1,22 +1,77 @@
 import * as bs58 from 'bs58';
 import * as crypto from 'crypto';
 import * as openpgp from 'openpgp';
-import Eddsa, { KeyShare } from '@bitgo/account-lib/dist/src/mpc/tss';
+import Eddsa, { KeyShare, JShare, SignShare, PShare, XShare, RShare, GShare } from '@bitgo/account-lib/dist/src/mpc/tss';
 
 import { BaseCoin, KeychainsTriplet } from '../baseCoin';
 import { Keychain } from '../keychains';
 import { BitGo } from '../../bitgo';
+import { KeyIndices, Memo, VerificationOptions, Wallet } from '..';
+import { RequestTracer } from './util';
+import _ = require('lodash');
+
+// #region Interfaces
+interface PrebuildTransactionWithIntentOptions {
+  reqId: RequestTracer;
+  intentType: string;
+  sequenceId?: string;
+  recipients: {
+    address: string;
+    amount: string | number;
+  }[];
+  comment?: string;
+  memo?: Memo;
+  verification?: VerificationOptions;
+}
+
+enum ShareKeyPosition {
+  USER = '1',
+  BACKUP = '2',
+  BITGO = '3',
+}
+
+// complete with more props if neccesary
+interface TransactionRequestJSON {
```

Comment:
thoughts on renaming this `TxRequest` to match the resource name from the url?
makes it clear that this model is what we get back from calling the `.../txRequests` endpoint